File: Ficheros texto_binario/EjemploConjunto.py
try:
    with open("fichero.txt", "w") as writer:
        writer.write("Texto a escribir\n")
        writer.write("Otra linea de texto\n")
except Exception as e:
    print(f"Error al escribir el archivo: {e}")

try:
    with open("fichero.txt", "r") as reader:
        for linea in reader:
            print(linea.strip())  
except Exception as e:
    print(f"Error al leer el archivo: {e}")

File: Hilos Java/Ficheros texto_binario/EjemploConjunto.java
import java.io.BufferedReader;
import java.io.BufferedWriter;
import java.io.FileReader;
import java.io.FileWriter;

public class EjemploConjunto {
    public static void main(String[] args) {
        try (BufferedWriter writer = new BufferedWriter(new FileWriter("fichero.txt"))){
            writer.write("Texto  a escribir\n");
            writer.write("Otra linea de texto");
        } catch (Exception e) {
            // TODO: handle exception
        }
        try (BufferedReader reader = new BufferedReader(new FileReader("fichero.txt"))){
            String cadena = reader.readLine();
            
            while (cadena != null) {
                System.out.println(cadena);
                cadena = reader.readLine();
                
            }
        } catch (Exception e) {
            // TODO: handle exception
        }
    }
}
